quickjs.c

```diff
@@ -50219,3 +50224,37 @@ void JS_AddIntrinsicTypedArrays(JSContext *ctx)
     JS_AddIntrinsicAtomics(ctx);
 #endif
 }
+
+/* Performance */
+
+static uint64_t js__now_ms(void)
+{
+    // TODO(saghul) Windows support.
+    struct timespec ts;
+    clock_gettime(CLOCK_MONOTONIC, &ts);
+    return (((uint64_t) ts.tv_sec) * 1000 + ts.tv_nsec / 1e6);
```

Comment:
```suggestion
    return (uint64_t) ts.tv_sec * 1000 + ts.tv_nsec / 1e6;
```
You could also write it as `return ts.tv_sec * 1000ull + ts.tv_nsec / 1e6;`